index.html

```diff
@@ -14,6 +14,76 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class='pageContainer'>
+    <header>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About</a>
+        <a href="#">Portfollio</a>
+        <a href="#">Contact</a>
+      </nav>
+    </header>
+    <section class='photoSection'>
+      <div class='photoContainer'>
+        <img class='profilePicture' src="/img/frank-106.png" alt="">    
```

Comment:
not seeing a piccture here unfortunately, though it may be because of html preview